.github/labeler.yml

```diff
@@ -0,0 +1,43 @@
+# Add/remove 'critical' label if issue contains the words 'urgent' or 'critical'
+critical:
+    - '(critical|urgent)'
+
+cli:
+    - 'cli'
```

Comment:
I think the regex is case-sensitive by default, so this would not match `CLI`. The action seems to support regex options (see https://github.com/github/issue-labeler/pull/15), so we could ignore the case.

Reply:
Added!